ff-get.js

```diff
@@ -69,6 +69,9 @@ function _reallyRead (args) {
   const ficInflate = use('fic-inflate')
   return ficInflate(deflatedFic, fetchAndSpin.withOpts({cacheBreak: false})).then(fic => {
     const filenameize = use('filenameize')
+    if (fic.words === 0) {
```

Comment:
I was going to say this wasn't safe due to subfics, but those actually only exist if you edit them into a file by hand as none of the site implementations support them.